docs/design/ovs-pipeline.md

```diff
@@ -406,42 +406,57 @@ In the future this table may support an additional mode of operations, in which
 it will implement kube-proxy functionality and take care of performing
 load-balancing / DNAT on traffic destined to services.
 
-### AntreaPolicyEgressRuleTables (45, 49)
+### AntreaPolicyEgressRuleTable (45)
 
-For these two tables, you will need to keep in mind the ACNP
+For this table, you will need to keep in mind the ACNP
 [specification](#antrea-networkpolicy-crd-implementation)
 that we are using.
 
-These two tables are used to implement the egress rules across all Antrea-native policies.
-Depending on the tier to which the ACNP or ANP belongs to, the rules will be 
+This table is used to implement the egress rules across all Antrea-native policies,
+except for policies that are created in the Baseline Tier. Antrea-native policies
+created in the Baseline Tier will be enforced after K8s NetworkPolicies, and their
+egress rules are installed in the EgressDefaultTable (60) and EgressRuleTable (50),
+respectively.
+Depending on the tier to which the ACNP or ANP belongs to, the egress rules will be 
 installed in a table corresponding to that tier. The egress table to tier mappings 
 is as follows:
 
 ```text
-Application tier -> DefaultTierEgressRuleTable(49)
-All other tiers  -> MultiTierEgressRuleTable(45)
+Baseline Tier   ->  EgressDefaultTable(60)
+All other Tiers ->  AntreaPolicyEgressRuleTable(45)
 ```
```

Comment:
I feel like this (and the 2 sentences preceding it) is redundant with what you explain above:

>This table is used to implement the egress rules across all Antrea-native policies,
except for policies that are created in the Baseline Tier. Antrea-native policies
created in the Baseline Tier will be enforced after K8s NetworkPolicies, and their
egress rules are installed in...

Reply:
rephrased to remove the redundancy